### DISCUSSION TITLE
Editor extensibility

### DISCUSSION THREAD

**Discussion by cbaal83** [nxrighthere/UnrealCLR#88] [Help] [answered]
Hi,
i would like to extent the Unreal Editor using an utility widget  and calling some c# code from there.

As of now, it looks like that the "Find managed Function" blueprint only seems to find functions during play-mode. 
Is there a workaround for this?

Best regards,
//cbaal

**Comment by nxrighthere** [comment, marked as answer]
Editor extensibility is a low-priority task for now. It's planned but for later versions.
